scripts/README.md

```diff
@@ -0,0 +1,64 @@
+products:
+- azure
+- azure-digital-twins
+description: "How to deploy Azure Digital twins using automated powershell scripts"
+---
+
+# Use Automated scripts to deploy Azure Digital Twins
+Deploy.ps1 is an Azure Digital Twins code sample used to deploy an Azure Digital Twins instance and set the required permissions. It can also be used as a starting point for writing your own scripted interactions.
+
+## Prerequisites
+
+* You'll need Azure subscription. You can set one up for free [here](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
+* You'll need a resource group in your Azure subscription. You can create your resource group using [this](https://docs.microsoft.com/azure/azure-resource-manager/management/manage-resource-groups-cli) article.
+* Download [Azure Digital Twins samples](https://docs.microsoft.com/samples/azure-samples/digital-twins-samples/digital-twins-samples/) to your machine. In the downloaded sample folder, the deployment script is located at _Azure_Digital_Twins_samples.zip > scripts > **deploy.ps1**_.
+* To be able to complete all the steps in this article, you need to be classified as an Owner in your Azure subscription.
+    * You can check your permission level by running this command in Cloud Shell:
+
+```azurecli-interactive
+az role assignment list --assignee <your-Azure-email>
+```
+
+If you are an owner, the `roleDefinitionName` value in the output is *Owner*:
+
+![Screenshot of user checking role owner](/media/scripts/owner-role.png)
+
+If you find that the value is *Contributor* or something other than *Owner*, you can proceed in one of the following ways:
+
+* Contact your subscription Owner and request the Owner to complete the steps in this article on your behalf.
+* Contact either your subscription Owner or someone with User Access Admin role on the subscription, and request that they elevate you to Owner on the subscription so that you will have the permissions to proceed yourself. Whether this is appropriate depends on your organization and your role within it.
+
+## Setup
+
+1. Login to [Azure Cloud Shell](https://shell.azure.com) by using the following command
+
+```azurecli
+az login
+```
+
+2. Upload the *deploy.ps1* file that you have downloaded earlier from your machine to the Azure cloud shell by selecting *Upload/Download* icon in the navigation bar.
+
+![Screenshot for uploading a file to Azure cloud shell](/media/scripts/cloud-shell-upload-file.png)
+
+## Running the sample
+
+deploy.ps1 script can be run in two modes.
+
+* *.\deploy.ps1*
+    This mode of the script:
+    * Creates an instance
+    * Sets up user access permissions
+    * Sets access permissions for client applications
+* *.\deploy.ps1 -endtoend*
+    This mode of the script:
+    * Sets up an Azure Digital Twins instance
+    * Uses an [Azure Functions](../azure-functions/functions-overview.md) app to route simulated telemetry from an [IoT Hub](../iot-hub/about-iot-hub.md) device into digital twin properties
```

Comment:
This isn't quite what the script does. The script itself doesn't use the functions app to route the telemetry -- it just creates the resources that you'll use later to do that in the tutorial. 
We should maybe list the resources that the script creates, and link to the article that's the next step (teaching them how to use the resources to do the route telemetry thing).

Reply:
Also, relative links like that IoT Hub one won't work here, because the file you're referencing is not in this repo. Need to give the full docs site link if you want to link out to other services from this readme